addons/l10n_in_ewaybill_stock/models/stock_move.py

```diff
@@ -0,0 +1,57 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+    _description = "Stock Move Ewaybill"
+
+    ewaybill_id = fields.Many2one(
+        comodel_name='l10n.in.ewaybill')
+
+    ewaybill_company_currency_id = fields.Many2one(
+        comodel_name='res.currency',
+        related='company_id.currency_id')
+
+    # Need to store values because we send it to the ewaybill and we need to keep the same value
+    ewaybill_price_unit = fields.Monetary(
+        compute='_compute_ewaybill_price_unit',
+        currency_field='ewaybill_company_currency_id',
+        store=True)
+    ewaybill_tax_ids = fields.Many2many(
+        comodel_name='account.tax',
+        string="Taxes",
+        compute='_compute_tax_ids',
+        store=True)
```

Comment:
```suggestion


```
Why this fear of changing value?  In the worst case, people will just check the json.